### DISCUSSION TITLE
Nested docs across collections

### DISCUSSION THREAD

**Discussion by jaschaio** [payloadcms/payload#4626] [Feature Requests & Ideas]
It seems that the nested docs plugin does not support a parent child relationship across collections.

The [example previously given](https://github.com/payloadcms/payload/issues/4367) was Organizations » Employees

Another one I can think of is Courses » Lessons

There was a [PR for it](https://github.com/payloadcms/plugin-nested-docs/pull/27) but it looks like it didn't survive the move to a monorepo.

**Comment by danielmlr-tiba** [comment]
any updates on this? would be a great feature

**Comment by HarleySalas** [comment]
Another vote for this...

Need to be able to localize all slugs. Having to hardcode a route for param based collections outside of the page collection is not flexible. Really hoping we can get to the point that we're including one [[...segments]] route and nothing else

**Comment by ainsleyclark** [comment]
Anyone know if this is still on the table?

**Reply by msclp** [reply]
there is [this](https://github.com/payloadcms/payload/pull/10833), but stale

**Comment by mabel-lawrence-dxd** [comment]
This would be very helpful for me as well. Have there been any updates on if this was or will be implemented?
